spec/grover/processor_spec.rb

```diff
@@ -353,13 +353,15 @@
             {
               'cookies' => [
                 { 'name' => 'grover-test', 'value' => 'nom nom nom', 'domain' => 'cookie-renderer.herokuapp.com' },
-                { 'name' => 'other-domain', 'value' => 'should not display', 'domain' => 'example.com' }
+                { 'name' => 'other-domain', 'value' => 'should not display', 'domain' => 'example.com' },
+                { 'name' => 'escaped', 'value' => '%26%3D%3D', 'domain' => 'cookie-renderer.herokuapp.com' }
               ]
             }
           end
 
-          it { expect(pdf_text_content).to include 'Request contained 1 cookie' }
+          it { expect(pdf_text_content).to include 'Request contained 2 cookies' }
           it { expect(pdf_text_content).to include '1. grover-test nom nom nom' }
+          it { expect(pdf_text_content).to include '2. escaped %26%3D%3D' }
```

Comment:
Ok so this was actually a bug on my part on the cookie render app I wrote.. see https://github.com/abrom/cookie_renderer/commit/b724fb04d6a051886bab0e10ea12cfbb812b7cf0

With that fixed the spec now works how you would expect it to (ie decoded..)

```suggestion
          it { expect(pdf_text_content).to include '2. escaped &==' }
```